container.go

```diff
@@ -273,9 +274,10 @@ func (c *container) NewTask(ctx context.Context, ioCreate cio.Creator, opts ...N
 	if info.RootFS != nil {
 		for _, m := range info.RootFS {
 			request.Rootfs = append(request.Rootfs, &types.Mount{
-				Type:    m.Type,
-				Source:  m.Source,
-				Options: m.Options,
+				Type:      m.Type,
+				Source:    m.Source,
+				Options:   m.Options,
+				MetaDatas: m.MetaDatas,
```

Comment:
This should be `Metadata` (or `MetaData`) (singular), as `data` is already plural.

But ok to wait with changing as the discussion hasn't completed yet, so maintainers may have more input on this 😅